ocs_ci/ocs/awscli_pod.py

```diff
@@ -69,6 +72,17 @@ def create_awscli_pod(scope_name=None, namespace=None, service_account=None):
         lambda: Pod(**get_pods_having_label(constants.S3CLI_LABEL, namespace)[0])
     )()
     wait_for_resource_state(awscli_pod_obj, constants.STATUS_RUNNING, timeout=180)
+
+    awscli_pod_obj.exec_cmd_on_pod(
+        f"cp {constants.SERVICE_CA_CRT_AWSCLI_PATH} {constants.AWSCLI_CA_BUNDLE_PATH}"
+    )
+
+    if storagecluster_independent_check() and config.EXTERNAL_MODE["rgw_secure"]:
+        log.info("Concatenating the RGW CA to the AWS CLI pod's CA bundle")
+        awscli_pod_obj.exec_cmd_on_pod(
+            f"bash -c 'wget -O - {config.EXTERNAL_MODE['rgw_cert_ca']} >> {constants.AWSCLI_CA_BUNDLE_PATH}'"
```

Comment:
@sagihirshfeld 
`>>` Question for this. Do we really want to just add content to existing content, so each run, it will add the data into same file, just want to confirm, it's intentional? Or if we wanted to just write it once `>` and replace previous content?

Reply:
Apologies for the late response here.

The use of `>>` here is intentional because I want to concatenate the rgw-ssl specific certificate to the existing one which was injected when the pod was created. 

This should only happen once per such bundle file  and not create a huge repeating file by accident, because `AWSCLI_CA_BUNDLE_PATH` is in ephemeral storage on the pod (under /tmp) and we're running this logic only once per such pod creation.

Reply:
@sagihirshfeld storing this AWSCLI_CA_BUNDLE_PATH in ephemeral storage (/tmp) is causing the regression. After node restart tests, these ca-cert data is lost causing failure.